sympy/parsing/c/c_parser.py

```diff
@@ -329,26 +349,18 @@ def transform_var_decl(self, node):
                         ))
 
             except StopIteration:
-
-                if (node.type.kind == cin.TypeKind.INT):
-                    type = IntBaseType(String('integer'))
-                    value = Integer(0)
-                elif (node.type.kind == cin.TypeKind.FLOAT):
-                    type = FloatBaseType(String('real'))
-                    value = Float(0.0)
-                elif (node.type.kind == cin.TypeKind.BOOL):
-                    type = Type(String('bool'))
-                    value = false
-                else:
-                    raise NotImplementedError("Only bool, int "
-                            "and float are supported")
+                return Variable(
+                node.spelling
+                ).as_Declaration(
+                    type = type
+                )
 
             return Variable(
                 node.spelling
-            ).as_Declaration(
-                type = type,
-                value = value
-            )
+                ).as_Declaration(
+                    type = type,
+                    value = value
+                )
```

Comment:
You can even have this return statement inside the try block, it might look a little more organized. But it doesn't make much difference tbh. You can leave it here too if you want to.

Reply:
Yeah :)